python/ray/serve/http_adapters.py

```diff
@@ -0,0 +1,59 @@
+from io import BytesIO
+from typing import List, Optional, Union
+
+from fastapi import File
+from pydantic import BaseModel, Field
+
+from ray.serve.model_wrappers import DataBatchType
+from ray.serve.utils import require_packages
+
+
+_1DArray = List[float]
+_2DArray = List[List[float]]
+_3DArray = List[List[List[float]]]
+_4DArray = List[List[List[List[float]]]]
+
+
+class NdArray(BaseModel):
+    """Schema for numeric array input."""
+
+    array: Union[_1DArray, _2DArray, _3DArray, _4DArray] = Field(
+        ...,
+        description=(
+            "The array content as a list or nested list. "
+            "You can pass in 1D to 4D array in its original form or the flatten verion."
```

Comment:
```suggestion
            "You can pass in a 1D to 4D array in its original form or the flattened verion."
```

Unclear what original form vs. flattened version means to me here. Give an example?

Reply:
it's just nested list, I will reword